src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -36,16 +36,21 @@ const propTypes = {
     /** Text to be shown in the tooltip */
     text: PropTypes.string.isRequired,
 
-    /** Number of pixels to set max-width on tooltip  */
-    maxWidth: PropTypes.number.isRequired,
-
     /** Maximum number of lines to show in tooltip */
     numberOfLines: PropTypes.number.isRequired,
+
+    /** Number of pixels to set max-width on tooltip  */
+    maxWidth: PropTypes.number,
+
+    /** Render custom content inside the tooltip. Note: This cannot be used together with the text props. */
```

Comment:
Should we throw error in console, if someone ens up using both?

Reply:
ah yes, good point. I think this was also brought up during the design doc process and I forgot to add the check here. Will add it!